File: benchmark/2026-09-17-7677581(latest)/semantic-compression/witnesses/swift/chunk_B/f09p2_V4_float_scalars.swift
var xs = [3.0, 1.0, 2.0]
let a = Double.nan
let b = 2.0
// BEGIN PROBE F09.P2
xs.sort(by: >)
let lt = a < b
// END PROBE F09.P2
print(xs, lt, b < a)

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/witnesses/java/chunk_B/f09p2_V4_floatscalars.java
import java.util.ArrayList;
import java.util.Comparator;
import java.util.List;

class Main {
    static String probe() {
        var xs = new ArrayList<>(List.of(2, 3, 1));
        double a = Double.NaN;
        double b = 9.0;
        // BEGIN PROBE F09.P2
        xs.sort(Comparator.reverseOrder());
        boolean lt = a < b;
        // END PROBE F09.P2
        return xs + " " + lt;
    }

    public static void main(String[] args) {
        System.out.println(probe());
    }
}
